Gemfile.lock

```diff
@@ -5,13 +5,9 @@ GIT
   specs:
     mimemagic (0.3.5)
 
-GEM
-  remote: https://rails-assets.org/
-  specs:
-    rails-assets-momentjs (2.22.2)
-
 GEM
   remote: https://rubygems.org/
+  remote: https://rails-assets.org/
```

Comment:
This change was happening to me as well. I am guessing it has something to do with whatever version of bundler you are on. See if updating your version of bundler takes this change away.